core/Azure.Mcp.Core/src/Areas/Server/Options/TransportTypes.cs

```diff
@@ -1,15 +1,23 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using System.Text.Json.Serialization;
+
 namespace Azure.Mcp.Core.Areas.Server.Options;
 
 /// <summary>
 /// Defines the supported transport mechanisms for the Azure MCP server.
 /// </summary>
-internal static class TransportTypes
+[JsonConverter(typeof(JsonStringEnumConverter))]
+public enum TransportTypes
```

Comment:
We should name this `TransportType`  (singular) as per [naming conventions](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-classes-structs-and-interfaces#naming-enumerations) for enum.

Reply:
make sense, I'll rename this one

Reply:
Reverted this back to a class (as in main). While testing learned that using an enum causes the common `System.CommandLine` issue where passing an invalid enum value results in an unfriendly InvalidOperationException. This might be the reason the original implementation avoided using an enum.